allennlp/training/trainer.py

```diff
@@ -260,17 +260,16 @@ def _train_epoch(self, epoch: int) -> dict:
 
         return self._get_metrics(train_loss, batch_num, reset=True)
 
-    def _should_stop_early(self, latest_metric: float, prev_metrics: List[float]) -> bool:
+    def _should_stop_early(self, prev_metrics: List[float]) -> bool:
```

Comment:
Given that this list now has the latest metric, `prev_metrics` doesn't make too much sense.  Maybe something like `metric_history`?  `metric_list`?  I don't have a great suggestion for a name here...